### ISSUE TITLE
Engg / EnginX next steps

### ISSUE THREAD

**Issue by FedeMPouzols** [mantidproject/mantid#13051]
This is a pseudo-umbrella issue to keep track of next priorities with EnginX support and Engg (Engineering diffraction) algorithms and interfaces in general. This could be useful to define priorities and divide up the work.

This should be done first:
- [x] [TRAC8368](http://trac.mantidproject.org/mantid/ticket/9788) - initial umbrella ticket - issue #9213 which would end when the first GUI becomes available.

Incremental additions that seem small and doable without side complications:
- [x] #13044 - Rename EnginX\* algorithms as Engg*
- [x] #13375 - Add focusing tab in the Engg GUI
- [x] #13486 - Extend focusing tab with "cropped" and "texture" focusing
- [x] #13808 - Support event mode
- [ ] TODO - next round of event mode support
- [ ] #13810 - Support multi-period mode
- [x] #13376 - Polish the GUI (includes MWRun widget, and output folders/file copies)
- [x] #13811 - improved 'plot focused data' 
- [x] #13783 - More GUI improvements: save focused results in multiple formats.
- [x] #13784 and #14835 - More GUI improvements: add multi-run focusing capability, with 2 modes: separately / sum of files
- [x] #13752 and #13643 - Capability to save .his files for GSAS
- [x] #13378 - Normalize by current + updated Vanadium corrections
- [x] #13045 - Add input property for rebinning bin size (for the initial rebinning in ToF)
- [x] #13050 - Fix EnginX instrument definition file
- [x] #15338 - split .prm file and change extension
- [ ] #12930 - fix units conversion when fitting focussed workspaces
- [ ] #13052 - Engg (EnggUtils): take bank numbers from grouping file
- [ ] #13381 - Fix transmission detectors in IDF

Bigger chunks of work that still need more discussion with scientists and that may span further work/issues/tickets:
- [ ] #12927 - Add a "peaks diagnosis" interface in the EnginX GUI
- [x] [TRAC9788](http://trac.mantidproject.org/mantid/ticket/9788) - Issue #10630 - GSAS integration
- All tickets/issues related to new additions to the GUI (too long list to add here):
  - [x] Single peaks fitting tab : #15906, #16005.
  - [ ] Many runs loading/saving capability for the peaks fitting tab.
  - [ ] Multi-fitting (GSAS-II) tab. #15552.

Newer requirements:
- [x]  #14591 - "cropped" calibration
- [x] #14590 - Provide a default full calibration
- [x] Some GUI improvements (disable error bars #14593, make advanced focusing disabled by default #14592)
- [x] #14594 - when calibrating, generate ob+..._South_bank.his files with the Vanadium pattern.
- [x] #14596 - additional system test with matching pairs of Vanadium and Ceria runs.
- [x] #14600 - default value for ExpectedPeaks visible in dialog (Ceria peaks)
- [x] #14788 - mask some bins out
- [x] #15541 - replace confusing working "spectrum IDs" with "spectrum numbers" for consistency.

**Comment by FedeMPouzols** [comment]
@Anders-Markvardsen: just FYI, I've stopped adding new issues in the old umbrella ticket which was becoming too bloated. It would end its life when the first GUI becomes available, as that looks like a good first milestone. This issue is now a compilation of future steps where we can keep track of the TODO items. I've assigned this to myself to keep it visible in my list, but feel free to edit/rearrange it as work on EnginX/Engg progresses.
